### PR TITLE
feat(component-index): 1655-component-index-no-results

### LINKED ISSUE #1655: Component Index: No results applied to filter
### Summary

- [ ] Apply no results to filter

- [ ] add right-padding to no results

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #1655 

### Summary

- [x] Apply no results to filter

- [x] add right-padding to no results 

- [x] Update "Website" to "Docs"

### CODE REVIEW

File: src/components/ComponentIndexPage/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-shadow */
```
Comment (joshblack) [inline src/components/ComponentIndexPage/index.js:1]:
```suggestion
```

**Review by joshblack — APPROVED** [review]
Looks great!

**Review by shinytoyrobots — APPROVED** [review]

**Review by johnbister — APPROVED** [review]
Looks great to me! My only suggestion would be a little more right padding so the text aligns closer with the paragraph above the index search. 
right padding

**Review by aagonzales — COMMENTED** [review]
I'm not seeing the right-padding that John mentions. Can we get that added in?

**Comment by andreancardona** [comment]
@aagonzales should be good now! Feel free to merge if it looks good to you

**Review by aagonzales — APPROVED** [review]
Perfect! thanks

**Pull request merged**